src/main/scala/cognite/spark/SdkV1Rdd.scala

```diff
@@ -79,7 +79,7 @@ case class SdkV1Rdd[A](
   def enqueueStreamResults(
       stream: Stream[IO, A],
       queue: EitherQueue,
-      processedItems: mutable.Set[Long],
+      processedItems: mutable.Set[String],
```

Comment:
Do we have to pass this? It's already in scope and it's a mutable, so passing the reference doesn't seem necessary.

Reply:
Seems like we do, at least according to IntelliJ. `enqueueStreamResults` is under `SdkV1Rdd`, not compute, to avoid the limit of methods > 50 lines.